tests/foreman/endtoend/utils.py

```diff
@@ -21,6 +22,8 @@ def client_provisioning(
             is available.
         :param package_name: Name of the package to be installed on the client.
         """
+        if not setting_is_set('clients'):
```

Comment:
Why not to use `skip_if_not_set`? Or if we need specific function for that purpose, why do we put it in `robottelo/decorators.py` module as it is not real decorator?